web/src/frontend/src/nav-soknad/components/timeoutbox/timeoutbox.less

```diff
@@ -0,0 +1,26 @@
+.modal__overlay {
```

Comment:
kanskje høre med Frode om han har noen tips til å få inn stylingen fra nav-frontend? Er veldig dumt å kopiere det rett derfra hvis det fungerer i utgangspunktet

Reply:
@ingvilin Er enig. Det er veldig dumt.

Reply:
Du må importere less fila i /index.less for at den skal komme med.